src/javascripts/legacy_app.js

```diff
@@ -1,8 +1,8 @@
 import BaseApp from 'base_app';
 
-// remove these if not needed
```

Comment:
wait, I didn't realise that this is the actual file we expect people to edit. My concern was that nobody will even open it to see the comment, but now I realise this makes sense as is.